2.0/0x300-Integrations/0x303-I3-Oracle.md

```diff
@@ -21,6 +21,13 @@ Category “I3” lists requirements related to the Oracle smart contract as one
 | **I3.4** | Verify that, when using Uniswap V3 TWAP as price oracle, liquidity is high enough and is distributed widely across most of the price range. |
 | **I3.5** | Verify that, the use a decentralized off-chain oracles unsusceptible to on-chain price manipulation attacks (e.g. Chainlink) is considered for low liquidity asset, ideally combining it with on-chain oracles to detect malicious values. |
 | **I3.6** | Verify that the value you are using has had enough time to be reported as invalid and has not been. |
+| **I3.7** | Verify that there is a check for stale prices when using Chainlink Oracle |
```

Comment:
Shouldn't this mention hearbeat? How do you know a price is stale?

Reply:
Yes indeed, the check should contain a heartbeat for the price feed but that is something more complex  for explanation, thats why I suppose that whoever read "verify that there is a stale price check" will already know the problem of this check more in depth. Also the article that I have added explains it if someone wants to check further. 

```solidity
require(block.timestamp - updatedAt <= heartbeatInterval,
  "ORACLE_HEARTBEAT_FAILED");       
```

That would be a good stale price check, including the correct heartbeat for the current price feed.
If you think that this "verify" statement can be written in a better and more understandable way, please correct it as you wish, I am totally okay with that and will be thankful.

Reply:
```suggestion
| **I3.7** | Verify that there is a check for stale prices when using Chainlink Oracle (including `hearbeat` check). |
```
I would simply mention that because the check does not need to explain everything but it would be nice if it mentioned the important things that user could do research on.